ethcore/src/evm/ext.rs

```diff
@@ -94,15 +94,15 @@ pub trait Ext {
 	fn extcodesize(&self, address: &Address) -> trie::Result<usize>;
```

Comment:
Really it should be an associated type `Error`. Externalities are lower level than the machine itself, so it doesn't make that much sense to return an `evm::Result` there.